TroopAnnouncements.html

```diff
@@ -65,15 +65,15 @@ <h2 class="w3-center">HTML slides</h2>
         
         <div class="w3-content" style="max-width:400px">
           <div class="mySlides w3-container w3-red">
-            <h1><b>Did You Know?</b></h1>
-            <h1><i>We plan to sell trips to the moon in the 2020s</i></h1>
+            <h1><b>Troop Outing</b></h1>
+            <h1><i>Our Next Troop Outing will be on Jan 13 2023, RSVP Via Scoutbook, or in person at next meeting.</i></h1>
           </div>
         
-          <img class="mySlides" src="https://media.discordapp.net/attachments/897388108591542304/1044398441981292634/b530938503bf4825a0a92b2f4be08296.png?width=393&height=1000" style="width:900%">
+          <img class="mySlides" src="https://imgur.com/qfDyUr0" style="width:900%">
         
           <div class="mySlides w3-container w3-xlarge w3-white w3-card-4">
-            <p><span class="w3-tag w3-red">New!</span>
-            <p>6 Crystal Glasses</p>
+            <p><span class="w3-tag w3-red">Merrit Bade University</span>
+            <p>We have been invited by  </p>
             <p>Only $99 !!!</p>
```

Comment:
> `<p>Only $99 !!!</p>`

This is not the right price for MBU.

Reply:
Im not sure where the 99 dollars came from.

Reply:
Price segment removed.